app/docs/dev/quickstart/universal.md

```diff
@@ -50,6 +50,10 @@ kumactl generate dataplane-token --name=redis > kuma-token-redis
 kumactl generate dataplane-token --name=app > kuma-token-app
 ```
 
+{% warning %}
+This action requires [authentication](/docs/{{ page.version }}/security/api-server-auth/#admin-user-token) unless executed against a control-plane running on localhost.
```

Comment:
Should we link here to `admin-user-token` or to the new [section](https://github.com/kumahq/kuma-website/pull/1125/files#diff-1c2e8d46cbcf07fec6d846d95887aca0049f0d3fab585724428e58dbccbcc5e6R37).

Reply:
I would link to the new section but link to `admin-user-token` in the new section above.

Reply:
I applied the suggestion and I think it's ok now.